### ISSUE TITLE
Downloading fails for some tiles

### ISSUE THREAD

**Issue by technodrome** [lovasoa/dezoomify-rs#35]
While for the vast majority of images in `zoomify` downloader works flawlessly, I have encountered many images at the following URL where download was not possible: `https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/ImageProperties.xml`. It just hangs infinitely with cmdline switch `-r 10`. When reduced to `-r 1`, it revealed some image groups being unavailable. This error very often happens at tiles 12/117 and 54/117 but that may be just a coincidence.

(`-r 10`)
```
[ETA:1s] ###################---------------------   54/117  Downloaded tile at x=256 y=1280
``` 
(`-r 1`)
```[ETA:2s] #####################-------------------   61/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-2-7.jpg: network error: HTTP status client error
[ETA:2s] ######################------------------   62/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-1-8.jpg: network error: HTTP status client error
[ETA:2s] ######################------------------   63/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-5-6.jpg: network error: HTTP status client error
[ETA:2s] ######################------------------   64/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-4-7.jpg: network error: HTTP status client error
[ETA:2s] #######################-----------------   65/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-7-6.jpg: network error: HTTP status client error
[ETA:3s] #######################-----------------   66/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-6-7.jpg: network error: HTTP status client error
[ETA:3s] ########################----------------   68/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-1-7.jpg: network error: HTTP status client error
[ETA:3s] ########################----------------   69/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-0-8.jpg: network error: HTTP status client error
[ETA:3s] ########################----------------   70/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-3-7.jpg: network error: HTTP status client error
[ETA:3s] #########################---------------   71/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-2-8.jpg: network error: HTTP status client error
[ETA:3s] #########################---------------   73/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-6-6.jpg: network error: HTTP status client error
[ETA:3s] ##########################--------------   75/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-7-7.jpg: network error: HTTP status client error
[ETA:3s] ##########################--------------   76/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-0-7.jpg: network error: HTTP status client error
[ETA:3s] ###########################-------------   77/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-3-8.jpg: network error: HTTP status client error
[ETA:3s] ###########################-------------   78/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-5-8.jpg: network error: HTTP status client error
[ETA:3s] ############################------------   79/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-0-9.jpg: network error: HTTP status client error
[ETA:3s] ############################------------   81/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-2-9.jpg: network error: HTTP status client error
[ETA:3s] #############################-----------   82/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-4-8.jpg: network error: HTTP status client error
[ETA:3s] #############################-----------   84/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-6-8.jpg: network error: HTTP status client error
[ETA:3s] ##############################----------   85/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-1-9.jpg: network error: HTTP status client error
[ETA:3s] ##############################----------   86/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-6-9.jpg: network error: HTTP status client error
[ETA:3s] ###############################---------   88/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-3-9.jpg: network error: HTTP status client error
[ETA:3s] ###############################---------   90/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-1-10.jpg: network error: HTTP status client error
[ETA:3s] ################################--------   91/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-0-10.jpg: network error: HTTP status client error
[ETA:3s] ################################--------   92/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-5-9.jpg: network error: HTTP status client error
[ETA:2s] #################################-------   94/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-7-9.jpg: network error: HTTP status client error
[ETA:2s] ##################################------   97/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-2-10.jpg: network error: HTTP status client error
[ETA:2s] ##################################------   98/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-0-11.jpg: network error: HTTP status client error
[ETA:2s] ##################################------   99/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-7-10.jpg: network error: HTTP status client error
[ETA:2s] ###################################-----  100/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-4-10.jpg: network error: HTTP status client error
[ETA:2s] ###################################-----  101/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-2-11.jpg: network error: HTTP status client error
[ETA:2s] ###################################-----  102/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-7-11.jpg: network error: HTTP status client error
[ETA:2s] ####################################----  103/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-6-10.jpg: network error: HTTP status client error
[ETA:2s] ####################################----  104/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-4-11.jpg: network error: HTTP status client error
[ETA:2s] ####################################----  105/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-1-11.jpg: network error: HTTP status client error
[ETA:1s] #####################################---  106/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-6-11.jpg: network error: HTTP status client error
[ETA:1s] #####################################---  107/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-3-11.jpg: network error: HTTP status client error
[ETA:1s] #####################################---  108/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-5-11.jpg: network error: HTTP status client error
[ETA:1s] ######################################--  109/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-8-11.jpg: network error: HTTP status client error
[ETA:1s] ######################################--  111/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-1-12.jpg: network error: HTTP status client error
[ETA:1s] #######################################-  112/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-3-12.jpg: network error: HTTP status client error
[ETA:1s] #######################################-  113/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-6-12.jpg: network error: HTTP status client error
[ETA:0s] ########################################  115/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-2-12.jpg: network error: HTTP status client error
[ETA:0s] ########################################  116/117  error with tile https://kramerius-vs.nkp.cz/search/zoomify/uuid:7894c8a0-42e8-11e4-af1d-001018b5eb5c/TileGroup0/3-5-12.jpg: network error: HTTP status client error
[ETA:0s] ########################################  117/117  Successfully downloaded 59 tiles out of 117
```

However, when the same image was URL is entered and downloaded through `dezoomify` webpage at `https://ophir.alwaysdata.net/dezoomify/dezoomify.html`, download works and the tiles were reassembled correctly.

What might be causing failure to download through dezoomify-rs when web version of the downloader works?

**Comment by lovasoa** [comment]
Thank you for the report. This is clearly a bug in dezoomify-rs, I'll investigate it.

**Comment by lovasoa** [comment]
I just released version 1.8.3, where I fixed this bug. It's currently being compiled, it'll be available on its [release page](https://github.com/lovasoa/dezoomify-rs/releases/tag/v1.8.3) when it's ready.

**Comment by technodrome** [comment]
Testing this now, so far works great! Fantastic work, thank you.
